spec/controllers/v0/occupations_controller_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'spec_helper'
+
+describe V0::OccupationsController do
+
+  describe '#index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.